helm/templates/statefulset.yaml

```diff
@@ -31,13 +28,45 @@ spec:
       volumes:
       - name: immudb-storage
         persistentVolumeClaim:
-          claimName: {{ include "immudb.fullname" . }}
+          claimName: immudb-claim
```

Comment:
All nodes in the cluster (both the leader and all replicas) will use the same persistent volume, immudb was not designed to handle such situation that could end up with data corruption.

For this purpose, each replica should have its own persistent volume assigned using [Volume Claim Template](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#volume-claim-templates)